packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -801,6 +803,13 @@ export interface IBuildImage {
  * If you need to use with an image that isn't in the named constants,
  * you can always instantiate it directly.
```

Comment:
This comment paragraph should be deleted (the constructor is now private).

Reply:
Done